polaris-react/src/components/Text/Text.tsx

```diff
@@ -43,9 +45,6 @@ const VariantFontWeightMapping: {[V in Variant]: FontWeight} = {
   heading2xl: 'semibold',
   heading3xl: 'semibold',
   heading4xl: 'bold',
-  bodySm: 'regular',
-  bodyMd: 'regular',
-  bodyLg: 'regular',
```

Comment:
I think we can get away with removing `font-weight: normal` here as it should use the default or parents value.